Package.swift

```diff
@@ -46,6 +56,23 @@ let package = Package(
                     resources: [
                         .copy("Resources/single_add.basic.ort")
                     ]),
+        .target(name: "OnnxRuntimeExtensions",
+                dependencies: ["onnxruntime_extensions", "onnxruntime"],
```

Comment:
If I recall correctly, I think the binary target name had to match the actual .xcframework name otherwise would get an error like:

```
xcodebuild: error: Could not resolve package dependencies:
  downloaded archive of binary target 'OnnxRuntimeCBinary' does not contain expected binary artifact named 'OnnxRuntimeCBinary'
  fatalError
```